monitoring/api/AlertSample/Program.cs

```diff
@@ -114,6 +136,21 @@ static void ListAlertPolicies(string projectId)
         }
         // [END monitoring_alert_list_policies]
 
+        static void ListNotificationChannels(string projectId)

```

Comment:
Do you have a region tag for this sample?

Reply:
Done.